bridge/docs/observability.md

```diff
@@ -0,0 +1,1422 @@
+- `withdraw_completed`: a withdraw has completed and received on the target stellar account.
+
+##### Bridge vault account related
+- `payment_received` : This event represents successful payment to the bridge account (a deposit).
+- `stellar_transaction_submitted` : This event represents successful transaction from the bridge account (a refund or a withdraw).
+
+### Metrics:
+
+This events describes a numeric measurement taken at given point in time.
+Metric events are often collected on a predictable frequency.
+
+#### Bridge vault account related
+- `wallet_balance`: This event describes a tft balance locked on the bridge and collected once a minute.
+
+### Log schema:
+
+#### Base fields
+The base field set contains all fields which are at the root of the events. These fields are common across all types of events.
+
+| Field | Type | Description | Required |
+| --- | --- | --- | --- |
+| level | string | yes | The level or severity of the log event |
+| version | number | yes |Log schema version |
+| source | object | yes | Information about the bridge instance |
+| event_action | string | yes | The action captured by the event |
+| event_kind | string | yes | Can gives high-level information about what type of information the event contains, without being specific to the contents of the event. One of `Event`, `Alert`, `Error`, `metric` |
+| metadata | object | no | Only present for logs that contain meta data about the event |
+| message | string | yes | The log message |
+| time | time | yes | The timestamp of the log event |
+| error | string | no | Only present for logs that contain an exception or error. The message of the exception or error |
+| category | string | no | One of `availability`, `persistency`, `stellar_monitor`, `tfchain_monitor`, `transfer`, `mint`, `refund`, `withdraw`, `vault` |
+| trace_id | string | no | Only present on business events. a unique identifier that is assigned to a trace, which represents a complete cross-chain transfer flow. |
+
+#### Categorization Fields
+##### event_kind field
+The value of this field can be used to inform how these kinds of events should be handled.
+
+- alert: This value indicates an event such as an alert or notable event.
+
+- event: This value is the most general and most common value for this field. It is used to represent events that indicate that something happened.
+
+- error: This value indicates that an error occurred during the operation of the bridge.
+
+- metric: This events describes a numeric measurement taken at given point in time.
+
+##### category field
+The value of this represents the "big buckets" of event categories
+
+- availability
+- persistency
+- stellar_monitor
+- tfchain_monitor
+- transfer
+- mint
+- refund
+- withdraw
+- vault
+
+#### source object
+the source field set contains all fields which are included in the source object, it is common across all types of events except of `bridge_init_aborted` error event.
```

Comment:
The source field set contains all fields which are included in the source object. It is common across all types of events except for `bridge_init_aborted` error event.